packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/WorkspaceSettings.tsx

```diff
@@ -305,13 +448,59 @@ export const WorkspaceSettings = () => {
           onSubmit={inviteLoading ? undefined : onInviteSubmit}
           css={{ display: 'flex', flexGrow: 1, maxWidth: 480 }}
         >
-          <UserSearchInput
-            inputValue={inviteValue}
-            allowSelf={false}
-            onInputValueChange={val => {
-              setInviteValue(val);
-            }}
-          />
+          <div style={{ position: 'relative', width: '100%' }}>
+            <UserSearchInput
+              inputValue={inviteValue}
+              allowSelf={false}
+              onInputValueChange={val => setInviteValue(val)}
+              style={{ paddingRight: 80 }}
+            />
+            {false /** TODO: UNCOMMENT WHEN BACKEND IS READY */ && (
```

Comment:
I got eager and implement this without the API 😅 

Still needs a few lines in the mutation after the API supports it